components/match2/commons/match_group_display_matchlist.lua

```diff
@@ -209,6 +216,22 @@ function MatchlistDisplay.Header(props)
 	return DisplayUtil.applyOverflowStyles(headerNode, 'wrap')
 end
 
+MatchlistDisplay.propTypes.DateHeader = {
+	dateHeader = 'table',
+}
+
+--[[
+Display component for a header in a matchlist.
+]]
+function MatchlistDisplay.DateHeader(props)
```

Comment:
Maybe `MatchlistDisplay.DateHeader({match = match})` and call `DisplayHelper.MatchCountdownBlock` from within `MatchlistDisplay.DateHeader`?